src/submission/MPI.jl

```diff
@@ -41,6 +41,11 @@ $FIELDS
     process. 
     """
     dependencies::Vector{Module} = []
+
+    """
+    Extra arguments passed to mpiexec.
+    """
+    mpiexec_args::String = ""
```

Comment:
I think it would be better to use the type `mpiexec_args::Cmd`, otherwise, if two arguments are passed they will not be tokenized properly I believe.